cypress/integration/stories/opa.spec.ts

```diff
@@ -0,0 +1,251 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {AdminSettingsPage} from '../../pages/admin-settings.po';
+import {ClustersPage} from '../../pages/clusters.po';
+import {ProjectsPage} from '../../pages/projects.po';
+import {WizardPage} from '../../pages/wizard.po';
+import {login, logout} from '../../utils/auth';
+import {Condition} from '../../utils/condition';
+import {Endpoint} from '../../utils/endpoint';
+import {RequestType, TrafficMonitor} from '../../utils/monitor';
+import {Preset} from '../../utils/preset';
+import {Datacenter, Provider} from '../../utils/provider';
+import {View} from '../../utils/view';
+import {WizardStep} from '../../utils/wizard';
+import * as _ from 'lodash';
+
+describe('OPA Story', () => {
+  const email = Cypress.env('KUBERMATIC_DEX_DEV_E2E_USERNAME_2');
+  const password = Cypress.env('KUBERMATIC_DEX_DEV_E2E_PASSWORD');
+  const projectName = _.uniqueId('e2e-test-project-');
+  const clusterName = _.uniqueId('e2e-test-cluster-');
+  const initialMachineDeploymentName = _.uniqueId('e2e-test-md-');
+  const initialMachineDeploymentReplicas = '1';
+  const constraintTemplateName = 'k8srequiredlabels'
+  const constraintTemplateSpec = atob('Y3JkOg0KICBzcGVjOg0KICAgIG5hbWVzOg0KICAgICAga2luZDogSzhzUmVxdWlyZWRMYWJlbHMNCiAgICB2YWxpZGF0aW9uOg0KICAgICAgIyBTY2hlbWEgZm9yIHRoZSBgcGFyYW1ldGVyc2AgZmllbGQNCiAgICAgIG9wZW5BUElWM1NjaGVtYToNCiAgICAgICAgcHJvcGVydGllczoNCiAgICAgICAgICBsYWJlbHM6DQogICAgICAgICAgICB0eXBlOiBhcnJheQ0KICAgICAgICAgICAgaXRlbXM6IHN0cmluZw0KdGFyZ2V0czoNCiAgLSB0YXJnZXQ6IGFkbWlzc2lvbi5rOHMuZ2F0ZWtlZXBlci5zaA0KICAgIHJlZ286IHwNCiAgICAgIHBhY2thZ2UgazhzcmVxdWlyZWRsYWJlbHMNCg0KICAgICAgdmlvbGF0aW9uW3sibXNnIjogbXNnLCAiZGV0YWlscyI6IHsibWlzc2luZ19sYWJlbHMiOiBtaXNzaW5nfX1dIHsNCiAgICAgICAgcHJvdmlkZWQgOj0ge2xhYmVsIHwgaW5wdXQucmV2aWV3Lm9iamVjdC5tZXRhZGF0YS5sYWJlbHNbbGFiZWxdfQ0KICAgICAgICByZXF1aXJlZCA6PSB7bGFiZWwgfCBsYWJlbCA6PSBpbnB1dC5wYXJhbWV0ZXJzLmxhYmVsc1tfXX0NCiAgICAgICAgbWlzc2luZyA6PSByZXF1aXJlZCAtIHByb3ZpZGVkDQogICAgICAgIGNvdW50KG1pc3NpbmcpID4gMA0KICAgICAgICBtc2cgOj0gc3ByaW50ZigieW91IG11c3QgcHJvdmlkZSBsYWJlbHM6ICV2IiwgW21pc3NpbmddKQ0KICAgICAgfQ==');

```

Comment:
This is no secret data, just the yaml for those fields (same for constraintSpec & gatekeeperConfig). Nevertheless I was wondering, if we want to store it somewhere else, maybe in a seperate file. WDYT?

Reply:
Separate file would be great.